recipe/meta.yaml

```diff
@@ -11,47 +10,64 @@ source:
   sha256: 4fd2eb3736cea503be94ab18d31a669addbaf2dd6a8249527cef94163bb93975
 
 build:
-  noarch: python
-  number: 3
-  script: python -m pip install . -vv
+  number: 4
 
-requirements:
-  host:
-    - hatchling
-    - pip
-    - python {{ python_min }}
+outputs:
+  - name: {{ name }}-base
+    script: build_base.sh
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python {{ python_min }}
+        - hatchling
+        - pip
+      run:
+        - python >={{ python_min }}
+        - geopandas >=1.0
+        - networkx >=2.5
+        - numpy >=1.22
+        - pandas >=1.4
+        - requests >=2.27
+        - shapely >=2.0
 
-  run:
-    - python >={{ python_min }}
+    test:
+      commands:
+        - python -m pip check
+      imports:
+        - osmnx
+      requires:
+        - pip
+        - python {{ python_min }}
 
-    # required dependencies' min versions from package's pyproject.toml
-    - geopandas >=1.0
-    - networkx >=2.5
-    - numpy >=1.22
-    - pandas >=1.4
-    - requests >=2.27
-    - shapely >=2.0
+  - name: {{ name }}
+    build:
+      noarch: python
+    requirements:
+      host:
+        - python {{ python_min }}
+      run:
+        - python >={{ python_min }}
```

Comment:
I thought I left a comment on this line but I don't see it now.

Doesn't this already come in via the `pin_subpackages` below?

Reply:
Yes it looks like you should be able to remove it.